app/models/tag.rb

```diff
@@ -368,6 +368,9 @@ def async(method, *args)
 
   # Class methods
 
+  def self.tag_feeds_key(id)
+    "/v1/rss_data_tags/#{id}"
```

Comment:
So the paths will not be seen by users.

We have v1 so if we need to change this code we don't need to worry about clearing the cache. I prefer having a format for cache keys.